troubleshooting-additional-digit-appended-to-apk.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Additional Digit Appended to Version Code after Android Build
+description: The Android Version Code has the number two appended to it after an application package cloud build.
```

Comment:
You can safely use 2 instead of two here. Alternatively, you can put 2 in brackets. I.e., the number two (2) appended